interpreter/cling/lib/Interpreter/IncrementalParser.cpp

```diff
@@ -911,8 +911,11 @@ namespace cling {
     PP.EnterSourceFile(FID, /*DirLookup*/nullptr, NewLoc);
     m_Consumer->getTransaction()->setBufferFID(FID);
 
-    if (!ParseOrWrapTopLevelDecl())
+    llvm::Expected<bool> res = ParseOrWrapTopLevelDecl();
```

Comment:
We should update the `ParseOrWrapTopLevelDecl` to return `llvm::Error` instead of `llvm::Expected<bool>`.

Reply:
Make the changes suggested. Please have a look.